RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate redux module roles to typescript

## Proposed changes

Migrate redux module roles to typescript

## Issue(s)	


## How to test or reproduce

Log into the app and check if the roles have been set correctly.
Change a role and check if it was changed correctly.
Delete a role and check if it was deleted correctly.

## Screenshots

## Types of changes




## Checklist



## Further comments
